runelite-client/src/main/java/net/runelite/client/plugins/objectindicators/ObjectIndicatorsOverlay.java

```diff
@@ -60,8 +61,11 @@ private ObjectIndicatorsOverlay(Client client, ObjectIndicatorsConfig config, Ob
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
-		for (TileObject object : plugin.getObjects())
+		for (ColorTileObject colorTileObject : plugin.getObjects())
 		{
+			TileObject object = colorTileObject.getTileObject();
+			Color color = colorTileObject.getColor();
```

Comment:
This should be checked for `null` and set to a real value before calling `renderPolygon`.